examples/components/wayback-search-component.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: wayback-search
   title: Wayback search
   description: Search of the wayback machine
+  annotations:
+    backstage.io/techdocs-ref: dir:../techdocs/wayback-search
+    pagerduty.com/integration-key: 1cf15c07f6f440e0a8d65b7ce80be834
```

Comment:
I'm pretty sure we need an API key for this integration to work

Reply:
https://nic-tester.enginehouse.dev/catalog/default/component/wayback-search/docs/ <- here is a link to the docs

Reply:
We decided to get a test pager duty account (I think there are free development accounts) with a token that it would be ok to use publicly

Reply:
My last understanding on this was that the product tour would have the user input the key in the settings as part of it so it is ok if it doesn't exist from the beginning.